MSProject/LibraryCollectionView/WBLibraryCollectionViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  WBLibraryCollectionViewController.swift
+//  MSProject
+//
+//  Created by Matias Spinelli on 11/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class WBLibraryCollectionViewController: UIViewController {
+
+    private let libraryCollectionView: WBLibraryCollectionView = WBLibraryCollectionView.loadFromNib()!
```

Comment:
This property can be just called `_view`, as it's going to be the view for this controller, and it will save you some space.